cmd/revad/svcs/httpsvcs/ocdavsvc/get.go

```diff
@@ -40,50 +42,51 @@ func (s *svc) doGet(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	ref := &storageproviderv0alphapb.Reference{
-		Spec: &storageproviderv0alphapb.Reference_Path{Path: fn},
+	sReq := &storageproviderv0alphapb.StatRequest{
+		Ref: &storageproviderv0alphapb.Reference{
+			Spec: &storageproviderv0alphapb.Reference_Path{Path: fn},
+		},
 	}
-	req := &storageproviderv0alphapb.StatRequest{Ref: ref}
-	res, err := client.Stat(ctx, req)
+	sRes, err := client.Stat(ctx, sReq)
 	if err != nil {
 		logger.Error(ctx, err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
-	if res.Status.Code != rpcpb.Code_CODE_OK {
-		logger.Println(ctx, res.Status)
+	if sRes.Status.Code != rpcpb.Code_CODE_OK {
+		logger.Println(ctx, sRes.Status)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
-	info := res.Info
+	info := sRes.Info
 	if info.Type == storageproviderv0alphapb.ResourceType_RESOURCE_TYPE_CONTAINER {
 		logger.Println(ctx, "resource is a folder, cannot be downloaded")
 		w.WriteHeader(http.StatusNotImplemented)
 		return
 	}
 
-	req2 := &storageproviderv0alphapb.InitiateFileDownloadRequest{
+	dReq := &storageproviderv0alphapb.InitiateFileDownloadRequest{
 		Ref: &storageproviderv0alphapb.Reference{
 			Spec: &storageproviderv0alphapb.Reference_Path{Path: fn},
 		},
 	}
 
-	res2, err := client.InitiateFileDownload(ctx, req2)
+	dRes, err := client.InitiateFileDownload(ctx, dReq)
 	if err != nil {
 		logger.Error(ctx, err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
-	if res.Status.Code != rpcpb.Code_CODE_OK {
-		logger.Println(ctx, res.Status)
+	if dRes.Status.Code != rpcpb.Code_CODE_OK {
```

Comment:
this was using the `res` from the previous request, which is wrong. I prefixed the variables for clarity and to prevent these copy paste errors.